From f27c02a09ed314256c163bb498ca9a013ff54efd Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 10 Mar 2009 10:23:26 +0800
Subject: [PATCH] ENGR00109264 ipuv3 : CSC not correct for PP task

Color space convertion is not correct for ipu pp task.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 drivers/mxc/ipu3/ipu_ic.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mxc/ipu3/ipu_ic.c b/drivers/mxc/ipu3/ipu_ic.c
index 7191f75..1d658cd 100644
--- a/drivers/mxc/ipu3/ipu_ic.c
+++ b/drivers/mxc/ipu3/ipu_ic.c
@@ -311,7 +311,7 @@ void _ipu_ic_init_pp(ipu_channel_params_t *params)
 		if ((out_fmt == YCbCr) || (out_fmt == YUV)) {
 			/* Enable RGB->YCBCR CSC */
 			_init_csc(IC_TASK_POST_PROCESSOR, RGB, out_fmt);
-			ic_conf |= IC_CONF_PP_CSC2;
+			ic_conf |= IC_CONF_PP_CSC1;
 		}
 	}
 	if ((in_fmt == YCbCr) || (in_fmt == YUV)) {
-- 
1.5.4.4

